gapic/ads-templates/tests/unit/gapic/%name_%version/%sub/test_%service.py.j2

```diff
@@ -1214,7 +1214,7 @@ def test_{{ method_name }}_rest_required_fields(request_type={{ method.input.ide
                     {% if req_field.field_pb.type == 9 %}
                     "{{ req_field.field_pb.default_value }}",
                     {% else %}
-                    {{ req_field.type.python_type(req_field.field_pb.default_value or 0) }},
+                    str({{ req_field.type.python_type(req_field.field_pb.default_value or 0) }}),
```

Comment:
This does not seem right, and it fails generated unit tests for compute

Reply:
Now that you mention it, yes, I think you're right. This "else" should be an "elseif" specific to bools. Yikes! That was quite an oversight on my part. Thanks for catching it.